bridge/qb/shared/main.lua

```diff
@@ -1,9 +1,10 @@
-local starterItems = require 'config.server'.starterItems
 local qbShared = require 'shared.main'
 
+local starterItems = require 'config.shared'.starterItems or {}
 ---@deprecated use starterItems in config/server.lua
 qbShared.StarterItems = {}
 
+assert(next(starterItems), 'starterItems is missing or incorrect in shared/config.lua')
```

Comment:
Is empty starter items not a valid entry?

Reply:
non existent starterItems isn't valid cause the loop will fail. Since we moved it from server to shared all the existing configurations will be broken

Reply:
It defaults to empty table in the above line if nil